cognite/client/data_classes/templates.py

```diff
@@ -154,11 +172,17 @@ def __init__(
         external_id: str = None,
         template_name: str = None,
         field_resolvers: Dict[str, FieldResolvers] = None,
+        data_set_id: Optional[int] = None,
+        created_time: int = None,
```

Comment:
not sure how int = None is different from having Optional[int], but otherwise looks ok :+1: